### ISSUE TITLE
Running into issues using jBox with event delegation

### ISSUE THREAD

**Issue by bjankord** [StephanWagner/jBox#37]
I'm trying to set up jBox to work with event delegation so the tooltips are "lazy loaded" / only init'd when the user goes to use them.

So far I have this [jsfiddle](https://jsfiddle.net/bjankord/r5cj85jm/), which partially works, but I noticed two things that seemed odd.

On the first mouseover event of a tooltip item, the tooltip does not display, on any mousever event afterwards, the tooltip displays fine.

Is there a way to have a tooltip display immediately when used inside of a event handler like I've got set up in the  [jsfiddle](https://jsfiddle.net/bjankord/r5cj85jm/) on the first mouseover event?

Second, I noticed that each time I hover over a item that has a tooltip, a new tooltip is created. This is probably related to how I have the event handler set up, but was wondering if there is some option not to create multiple instances in this case.

**Comment by zepernick** [comment]
I just did something similar and got it working.  The element needs to have the class "jbox-tooltip"

```js
	$(document).on("mouseenter", ".jbox-tooltip:not(.jbox-init)", function() {
		var target = $(this);
		target.addClass("jbox-init");
		var jb = new jBox("Tooltip", {
				attach: target,
				trigger: "mouseenter",
				getTitle: "data-jbox-title",
				getContent: "data-jbox-content",
				fade: 500,
				closeOnMouseleave: true,
				constructOnInit: true
				
			});
		
		//trigger the event we just attached to
		target.mouseenter();
	});
```

**Comment by StephanWagner** [comment]
Sorry, I've been neglecting jBox for a while, just released 0.4.0 and going through the issues...

Always remember that you only should create one jBox instance for a specific purpose.

In your fiddle you create a new jBox instance each time you move your mouse into the element. Here check out this fiddle: http://jsfiddle.net/StephanWagner/r5cj85jm/5/

Note that you create a jBox on domready and reference it in a global variable. This jBox attaches itself to all $('.tooltip') elements. When you create more elements you can just reattach you jBox to the new elements. This way you won't clutter your DOM
